moneo.py

```diff
@@ -95,6 +95,7 @@ def check_insights_config(args, parser):
     parser.add_argument('-i', '--insights', action='store_true', help='Experimental feature: Enable exporting of metrics to Azure Insights. Requires a valid instrumentation key and base_url for the Prometheus DB in config.ini')
     parser.add_argument('type', metavar='type', type=str, default='full', nargs="?", help='Type of deployment/shutdown. Choices: {manager,workers,full}. Default: full.')
     parser.add_argument('-p', '--profiler_metrics', action='store_true', default=False, help='Enable profile metrics (Tensor Core,FP16,FP32,FP64 activity). Addition of profile metrics encurs additional overhead on computer nodes.')
+    parser.add_argument('-f', '--fork_processes', default=16, type=int, help='The number of processes used to deploy/shutdown/update Moneo. Increase this value to for a higher process count. Default is 16.')
```

Comment:
Should we say anything about why you might want to increase the processor count here? Or maybe in a readme file?

Reply:
Ignore the multi-line question, it looks you address that in the other PR.

Reply:
I think I can update the help and readme usage

Reply:
addressed